backend/src/test/java/com/redhat/cloud/notifications/routers/OrgConfigResourceTest.java

```diff
@@ -127,6 +134,8 @@ void testInsufficientPrivileges() {
         given()
             .header(noAccessIdentityHeader)
             .when()
+            .contentType(MediaType.APPLICATION_JSON)
```

Comment:
Body is not requested to check 403.
Then we can validate checks order: first secutity, then content format.

Reply:
Good point, yes. I moved the validation checks to the `internal*` functions so that we keep the authorization check first.

Reply:
Actually I need to review it because that does not work as expected. I'm figuring out how to do it.

Reply:
All right, fixed!